code/config.py

```diff
@@ -305,6 +306,30 @@ def load_input_paths(paths):
     return paths
 
 
+def cleaned_load_profiles_paths(paths, param):
```

Comment:
why not within load_paths?

Reply:
Wanted to keep them separate, i'll add them in load_paths then